teamcity: failed tests on master: testrace/TestReplicaNotLeaseHolderError

The following tests appear to have failed:

[#666518](https://teamcity.cockroachdb.com/viewLog.html?buildId=666518):


```
--- FAIL: testrace/TestReplicaNotLeaseHolderError (0.090s)
replica_test.go:900: cannot replace lease repl=(n1,s1):1 seq=1 start=0.000000000,0 exp=1.800000124,3 pro=0.900000124,4 with repl=(n2,s2):2 seq=2 start=0.900000123,5 exp=0.900000134,0: proposed under invalid lease
--- FAIL: testrace/TestReplicaNotLeaseHolderError (0.110s)
replica_test.go:900: cannot replace lease repl=(n1,s1):1 seq=1 start=0.000000000,0 exp=1.800000124,3 pro=0.900000124,4 with repl=(n2,s2):2 seq=2 start=0.900000123,5 exp=0.900000134,0: proposed under invalid lease
------- Stdout: -------
I180518 19:11:52.438885 12514 storage/store.go:1443  [s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/replica_test.go
+++ b/pkg/storage/replica_test.go
@@ -879,10 +879,15 @@ func TestReplicaLease(t *testing.T) {
 
 func TestReplicaNotLeaseHolderError(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	tc := testContext{}
+
 	stopper := stop.NewStopper()
 	defer stopper.Stop(context.TODO())
-	tc.Start(t, stopper)
+
+	tc := testContext{manualClock: hlc.NewManualClock(123)}
+	cfg := TestStoreConfig(hlc.NewClock(tc.manualClock.UnixNano, time.Nanosecond))
+	cfg.TestingKnobs.DisableAutomaticLeaseRenewal = true
+	tc.StartWithStoreConfig(t, stopper, cfg)
+
 	secondReplica, err := tc.addBogusReplicaToRangeDesc(context.TODO())
 	if err != nil {
 		t.Fatal(err)

